src/TConsole.cpp

```diff
@@ -1069,6 +1069,10 @@ void TConsole::setConsoleBgColor(int r, int g, int b, int a)
 
 void TConsole::scrollDown(int lines)
 {
+    if (!mScrollingEnabled) {
```

Comment:
```suggestion
    if ((mType & (UserWindow|SubConsole)) && !mScrollingEnabled) {
```

Reply:
Oops, the same change is also desired by me on the `scrollUp(...)` method.

Reply:
Applied.